aten/src/ATen/Storage.h

```diff
@@ -31,10 +31,12 @@ struct AT_API Storage {
   bool resizable() const { return storage_impl_->resizable(); }
   // get() use here is to get const-correctness
   void* data() const { return storage_impl_.get()->data(); }
+  const at::DataType& dtype() const {
```

Comment:
Return this by value, not by reference.

Reply:
Will change, I got this from [caffe2](https://github.com/pytorch/pytorch/blob/master/caffe2/core/storage.h#L54).